src/infrastructure/NodeService.js

```diff
@@ -114,7 +114,7 @@ class NodeService {
             nodeInfo.roles === 3 ||
             nodeInfo.roles === 6 ||
             nodeInfo.roles === 7
-            	? 'http://' + nodeInfo.host + ':' + (globalConfig.apiNodePort || 3000)
+            	? 'https://' + nodeInfo.host + ':' + (globalConfig.apiNodePort || 3001)
```

Comment:
you may need to 'ping' the community node to know if they have migrated to HTTPS 3001 or they are still in HTTP 3000

Reply:
Yes, this is formatting the default data when it returns for the statistic service or node peers.
I assume most of the nodes will be enabled `https`.

It will ping the node when the user visits `node/<publicKey>` in explorer.